distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP.enso

```diff
@@ -79,7 +80,10 @@ type HTTP
        - follow_redirects: Whether or not the client should follow redirects.
        - proxy: The proxy that the client should use, if any.
        - version: The HTTP version supported by the client.
-    Value timeout follow_redirects proxy version
+       - custom_ssl_context: A custom SSL context to use for requests, or
+         Nothing if the default should be used. For most use cases, it is
+         recommended to use the default.
+    Value timeout follow_redirects proxy version custom_ssl_context
```

Comment:
I thought it is not needed, because this is an internal constructor - users should never use it anyway.

IMHO it is better to keep without the default, because:
- if a library dev writes a new function that creates a modified HTTP instance, if there's a default for this property, they may forget to set this argument in the constructor, and such a new function will then be losing the ssl context settings by accident. Such situation may not be caught by tests, because it's a scenario unlikely to be tested (we don't usually test that all properties are retained after modifying one). 
- If there's no default, forgetting about this property will return a not-fully-applied constructor, which is a clear mistake and will be immediately caught as the function will just not work correctly.

So IMO it is safer to _not have_ a default, because in most cases the context _should_ be set to inherit the previous one when modifying the HTTP instance.

The only argument for having it would be for user convenience - but this constructor is not supposed to be used by users anyway - we have `HTTP.new` for that.